rust/src/transformations/make_stable_expr/expr_col/mod.rs

```diff
@@ -0,0 +1,115 @@
+use polars::prelude::*;
+
+use crate::core::{Function, MetricSpace, StabilityMap, Transformation};
+use crate::domains::ExprDomain;
+use crate::error::*;
+
+use super::DatasetOuterMetric;
+
+/// Make a Transformation that returns a `col(column_name)` expression for a Lazy Frame.
+///
+/// # Arguments
+/// * `input_domain` - Domain of the expression to be applied.
+/// * `input_metric` - The metric under which neighboring LazyFrames are compared.
+/// * `col_name` - The name of the column to be selected.
+pub fn make_expr_col<M>(
+    input_domain: ExprDomain,
+    input_metric: M,
+    expr: Expr,
+) -> Fallible<Transformation<ExprDomain, ExprDomain, M, M>>
+where
+    M: DatasetOuterMetric,
+    M::Distance: Clone + 'static,
+    (ExprDomain, M): MetricSpace,
+{
+    let Expr::Column(col_name) = expr else {
+        return fallible!(MakeTransformation, "Expected Column expression");
+    };
+    let col_name = col_name.to_string();
+
+    let mut output_domain = input_domain.clone();
+    output_domain
+        .frame_domain
+        .series_domains
+        .retain(|v| v.field.name == col_name);
+
+    // ensure that column exists in the domain
+    output_domain.active_series()?;
```

Comment:
```suggestion
    output_domain.check_one_column()?;
```

Looking at the definition, it looks like there's a separate function for just the checking? Better to use a function that is explicitly a check, rather than relying on a side-effect.